### ISSUE TITLE
What is the best practice while subscribing to events

### ISSUE THREAD

**Issue by YogeshLele** [rs/pushd#58]
I have written a chat application in PHP which uses Pushd to send new message notification. When the user registers for the first time, I have created an event which is unique to that user and everytime when I have to send the push notification to that user I send the message to that event.

This scenario is fine in case of one to one chat. But when it comes to Group chat, Should I create a separate event for that group Or intelligently find out the events corresponding to recipients and push the message?

In short, what is the best practice- Use minimum number of events or Use minimum number of subscriptions per user?
